.github/workflows/publish.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Run Tests
         run: pnpm run cover 
 
-      - name: Publish Changeset and Package to NPM
+      - name: Publish packages
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
is the env var name right?

Reply:
So itwin/presentation has this: ` NODE_AUTH_TOKEN: ${{ secrets.NPMJS_PUBLISH_ITWIN }}`. I'm not sure if we can use the same

Reply:
no, you will want to use the same secret name. `NPMJS_PUBLISH_ITWIN`
this is an org level secret which is given to some repos
it will be the same as presentation repo